plugins/main/server/controllers/wazuh-api.ts

```diff
@@ -87,6 +84,7 @@ export class WazuhApiCtrl {
           idHost,
         );
       }
+      console.log('HELLO2');
```

Comment:
Perhaps this can be removed

Reply:
Removed here: cab4219a27a63e44fd0e77755f8009cd41b46179